script/participant/survey.js

```diff
@@ -0,0 +1,26 @@
+const { openBrowser, goto, inputField, write, click, above, below, image, toRightOf, prompt, to, fileField, attach, closeBrowser } = require('taiko');
+(async () => {
+    try {
+        await openBrowser();
+        await goto("http://localhost:3060");
+        await write("2223334444", inputField({ placeholder: "El numero del telefono" }));
+        await write("password", inputField({ placeholder: "Contraseña" }));
+        await click("Iniciar sesión");
+        await click("Control del medicacion");
+        await click("Sí");
+        await write("06:00 AM", inputField(above("Confirmar")));
+        await click("Confirmar");
+        await click("Sí");
+        await click("Náuseas o vómitos");
+        await click(image(below("Náuseas o vómitos")));
+        await write("Numbness", inputField(above("Confirmar")));
+        await click("Confirmar");
+        await click("Haga clic aquí para tomar una foto.");
+        await attach("src/images/strip_report.jpg", fileField());
```

Comment:
This tests the fallback for manually attaching a photo.
It works correctly in the UI, but the image is never sent to the database.